src/charts/data-manager.js

```diff
@@ -176,14 +177,18 @@ export default function DataManager () {
       .keys(dataBySeries.map(getID))
       .value((d, key) => d[key] || 0)
       .order(d3.stackOrderNone)
-      .offset(d3.stackOffsetNone)
+      .offset(d3[_stackOffset])
 
     // get stack totals
-    const allKeyTotals = dataByKey.map(d => ({
-      key: d[keys.KEY],
-      total: d3.sum(d[keys.SERIES].map(dB => dB[keys.VALUE])),
-      ...(d[keys.SERIES][0][keys.COUNTVAL] && {countval: d3.sum(d[keys.SERIES].map(dB => dB[keys.COUNTVAL]))})
-    }))
+    const allKeyTotals = dataByKey.map(d => {
+      const extent = d3.extent(d[keys.SERIES].map(dB => dB[keys.VALUE]))
```

Comment:
Computing total when you have negative values will not work with a simple sum, it needs to be an extent.

Reply:
This way you've done it is probably cleaner anyway, but just noting that you could probably still use sum if you mapped the values with Math.abs

Reply:
Mmmh, in fact the way I did it is wrong. I don't remember why I used abs before and changed it to extent, but I'm pretty sure it's wrong. Changing.